cpp/ps0005/main (5).cpp

```diff
@@ -2,48 +2,59 @@
 using namespace std;
 int main()
 {
+    // prompting user for 2 operands and an operator 1.5/2
+    double num1;
+    double num2;
+    char op;
+    double res;
     
-    int A;
-    int B;
-    char C;
-    float D;
+    cout << "enter the two operands \n";
+    cin >> num1 >> num2;
     
-   cout<<"enter your A , B \n";
-    cin>>A>>B;
-    
-   cout<<"enter your operator + - * / % \n";
-    cin>>C;
-    
-   switch(C)
+    cout << "enter your operator + - * / % \n";
+    cin >> op;
```

Comment:
- [x] declared variables to store the user input
- [x] chose suitable data types for the variables
- [x] prompted user to enter data
- [x] used `std::cin` in a right way to get user's input